src/main/java/com/polishuchenko/bookstore/repository/user/UserRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.polishuchenko.bookstore.repository.user;
+
+import com.polishuchenko.bookstore.model.User;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Long> {
+    Optional<User> findByEmail(String email);
```

Comment:
If I am not mistaken, it may be a lazy initialization here.

Reply:
You mean LazyInitializationException? Strange, it never happened